File: Programacion I/Programacion 1 Python/Clase07/Ejercicio3.py
# Ejercicio 7 - Calular Estacion del año

mes = int(input("Ingrese un numero (del 1 al 12) para calcular la estacion del año en la que se encuentra el mes: "))
estacion = None
if 1 <= mes <= 3:
    estacion = "Verano"
elif 4 <= mes <= 6:
    estacion = "Otoño"
elif 7 <= mes <= 9:
    estacion = "Invierno"
elif 10 <= mes <= 12:
    estacion = "Primavera"
else:
    print("El numero ingresado no esta dentro del rango valido de 1 a 12")
print(f"El mes {mes} corresponde a la estacion: {estacion}") if estacion != None else print("Vuelva a empezar")

File: Programacion I/Programacion 1 JS/Clase 10/Ejercicio3.js
//Calular la estacion del año con estructura switch

let mes = 1;
let estacion;
let condicion = mes <= 12 && mes > 0;

switch (mes) {
  case 1:
  case 2:
  case 3:
    estacion = 'Verano';
    break;
  case 4:
  case 5:
  case 6:
    estacion = 'Otoño';
    break;
  case 7:
  case 8:
  case 9:
    estacion = 'Invierno';
    break;
  case 10:
  case 11:
  case 12:
    estacion = 'Primavera';
    break;
  default:
    estacion = 'Mes invalido';
    break;
}

condicion
  ? console.log(`El mes ${mes} corresponde a la estacion: ${estacion}`)
  : console.log(estacion);
